backends/cuda/ceed-cuda-basis.c

```diff
@@ -134,21 +134,31 @@ extern "C" __global__ void grad(const CeedInt nelem, const int transpose, const
   }
 }
 
-extern "C" __global__ void weight(const CeedScalar * __restrict__ qweight1d, CeedScalar * __restrict__ v) {
-  CeedInt pre = BASIS_NQPT;
-  CeedInt post = 1;
-  for (CeedInt d=0; d<BASIS_DIM; d++) {
-    pre /= BASIS_Q1D;
-    for (CeedInt i=0; i<pre; i++) {
-      for (CeedInt j=0; j<BASIS_Q1D; j++) {
-        for (CeedInt k=0; k<post; k++) {
-          v[(i*BASIS_Q1D + j)*post + k] = qweight1d[j] * (d == 0 ? 1 : v[(i*BASIS_Q1D + j)*post + k]);
-        }
-      }
-    }
-    post *= BASIS_Q1D;
+// extern "C" __global__ void weight(const CeedScalar * __restrict__ qweight1d, CeedScalar * __restrict__ v) {
```

Comment:
Once we get ready to mege, it would be nice to clean out any old commented code. I know the OCCA backend has a lot of commented out code, but I think its preferable to delete lines that aren't going to be used. (I realize this is a WIP but I thought this was as good of a place to mention it as any)

Reply:
I kept this one for legacy reasons, I implemented a quick version without thinking too much about performance.

Reply:
I bring it up because what I've seen in the OCCA backend is that commented code does not get maintained and becomes dated and confusing quickly.